src/Api/AdminConsole/Public/Controllers/MembersController.cs

```diff
@@ -243,7 +243,7 @@ public async Task<IActionResult> Delete(Guid id)
         {
             return new NotFoundResult();
         }
-        await _organizationService.DeleteUserAsync(_currentContext.OrganizationId.Value, id, null);
+        await _organizationService.RemoveUserAsync(_currentContext.OrganizationId.Value, id, null);
```

Comment:
💭 Do we have plans to update the Public API endpoint as well?

Reply:
Great point, yes! I've renamed that endpoint too.

Reply:
💭 This didn't change the endpoint itself (which is fine, just pointing it out), I'm assuming future changes there will require some finesse. 
cc @eliykat